docs/src/modules/context.md

```diff
@@ -3,528 +3,34 @@
-
-
-### setBlockCoinbase
-
-*Sets the block coinbase to `who`.*
-
-
-```solidity
-function setBlockCoinbase(address who) internal returns (Context);
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`who`|`address`|The address to set as the block coinbase.|
-
-**Returns**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`<none>`|`Context`|The same context to allow function chaining.|
-
-
-### expectRevert
-
-*Function used to check whether the next call reverts or not.*
-
-
-```solidity
-function expectRevert(bytes memory revertData) internal;
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`revertData`|`bytes`|The function call data that that is expected to fail.|
-
-
-### expectRevert
-
-*Function used to check whether the next call reverts or not.*
-
-
-```solidity
-function expectRevert(bytes4 revertData) internal;
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`revertData`|`bytes4`|The function call signature that that is expected to fail.|
-
-
-### expectRevert
-
-*Function used to check whether the next call reverts or not.*
-
-
-```solidity
-function expectRevert() internal;
-```
-
-### expectEmit
-
-*Checks if an event was emitted with the given properties.*
-
-
-```solidity
-function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData) internal;
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`checkTopic1`|`bool`|Whether to check the first topic match.|
-|`checkTopic2`|`bool`|Whether to check the second topic match.|
-|`checkTopic3`|`bool`|Whether to check the third topic match.|
-|`checkData`|`bool`|Whether to check the data field match.|
-
-
-### expectEmit
-
-*Checks if an event was emitted with the given properties.*
-
-
-```solidity
-function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData, address emitter) internal;
-```
-**Parameters**
-
-|Name|Type|Description|
-|----|----|-----------|
-|`checkTopic1`|`bool`|Whether to check the first topic match.|
-|`checkTopic2`|`bool`|Whether to check the second topic match.|
-|`checkTopic3`|`bool`|Whether to check the third topic match.|
-|`checkData`|`bool`|Whether to check the data field match.|
-|`emitter`|`address`|The address of the expected emitter.|
-
-
-### mockCall
-
-*Function to mock a call to a specified address.*
-
-
-```solidity
-function mockCall(address callee, bytes memory data, bytes memory returnData) internal;
-```
-**Parameters**
+        // Enable/disable gas metering
+        ctx.pauseGasMetering();
+        ctx.resumeGasMetering();
 
-|Name|Type|Description|
-|----|----|-----------|
-|`callee`|`address`|The address for which the call should be mocked.|
-|`data`|`bytes`|The data for which the call should be mocked.|
-|`returnData`|`bytes`|The data that should be returned if `data` matches the provided call data.|
+        // Check if the current call is static
+        if (ctx.isStaticCall()) {
```

Comment:
```suggestion
        if (ctx.isStaticcall()) {
```

Maybe it is not necessary to include this one, it doesn't have much use inside a top level test